Update proposal process with new links (#41)
jupyter-governance/jupyter-foundation-governing-board: doc/funding/process.md

- **[This proposal template](https://docs.google.com/document/d/101cNUZbGCWUMn8LbDiSRjL5Q78e_31gN4-FqCiSp47o/edit?usp=drive_link)** has all the information needed for a new proposal. Authors duplicate it to do their writing in a public space.
- **[This proposals tracker Board](https://github.com/orgs/jupyter-governance/projects/13)** is where all proposals are tracked - update the "status" field as you write.
- **[](#timing-goals)**

## Overview of the process

1. **Discussion**: Create an issue to track discussion and status around the proposal.

   - [Issue template for new proposals →](https://github.com/jupyter-governance/funding-proposals/issues/new?template=proposal.md)
   - [Board for tracking all proposals →](https://github.com/orgs/jupyter-governance/projects/13)
1. **Writing**: When the idea is concrete enough, draft language for the proposal.
   - [New proposal template →](https://docs.google.com/document/d/101cNUZbGCWUMn8LbDiSRjL5Q78e_31gN4-FqCiSp47o/edit?usp=sharing).
   - Copy the template, put it in the same folder as the template so anyone can access.
   - Set the issue status to {kbd}`Writing`.
   - See [](#tips-for-writing-a-proposal).

> 1. **Submission**: When the proposal is ready for review, announce that it's ready for a subcommittee to take a look.
>    - Ping the [`@jupyter-foundation-governing-board` team](https://github.com/orgs/jupyter-governance/teams/jupyter-foundation-governing-board) in a comment on the proposal tracking issue.
>    - Set the status to {kbd}`Review`.

Reviewer: ```suggestion
1. **Submission**: When the proposal is ready for review, announce that it's ready for a subcommittee to take review:
   - Make a comment on the proposal issue pinging [`@jupyter-foundation-admins` team](https://github.com/orgs/jupyter-governance/teams/jupyter-foundation-admins) and telling them that the proposal should be considered as submitted.
   - Set the status in the proposal document to {kbd}`Reviewing`.
   - The foundation admins will also change the project tracking board status to indicate the proposal is being reviewed.
```

> Submissions are accepted on a rolling basis throughout the year, with funds being released according to our [budget allocations](./budget.md). 

Reviewer: ```suggestion
```

I think rely on the text below, i.e., we're doing a round, learning from it, and there will be more.

## Important links

- [`jupyter-governance/funding-proposals`](https://github.com/jupyter-governance/funding-proposals) is the repository where we track proposals.
- [This issue template](https://github.com/jupyter-governance/funding-proposals/issues/new?template=proposal.md) begins a new proposal idea, we use it to coordinate writing, discussion, and status.
- **[This proposal template](https://docs.google.com/document/d/101cNUZbGCWUMn8LbDiSRjL5Q78e_31gN4-FqCiSp47o/edit?usp=drive_link)** has all the information needed for a new proposal. Authors duplicate it to do their writing in a public space.

> - **[This proposals tracker Board](https://github.com/orgs/jupyter-governance/projects/13)** is where all proposals are tracked - update the "status" field as you write.

Reviewer: ```suggestion
- **[This proposals tracker Board](https://github.com/orgs/jupyter-governance/projects/13)** is where all proposals are tracked - update the "status" field as you write.
- **[](#timing-goals)**
```

## Overview of the process

1. **Discussion**: Create an issue to track discussion and status around the proposal.

   - [Issue template for new proposals →](https://github.com/jupyter-governance/funding-proposals/issues/new?template=proposal.md)
   - [Board for tracking all proposals →](https://github.com/orgs/jupyter-governance/projects/13)

> 1. **Writing**: When the idea is concrete-enough, draft language for the proposal.

Reviewer: ```suggestion
1. **Writing**: When the idea is concrete enough, draft language for the proposal.
```

   - [New proposal template →](https://docs.google.com/document/d/101cNUZbGCWUMn8LbDiSRjL5Q78e_31gN4-FqCiSp47o/edit?usp=sharing).
   - Copy the template, put it in the same folder as the template so anyone can access.
   - Set the issue status to {kbd}`Writing`.
   - See [](#tips-for-writing-a-proposal).
1. **Submission**: When the proposal is ready for review, announce that it's ready for a subcommittee to take a look.

>    - Ping the [`@jupyter-foundation-governing-board` team](https://github.com/orgs/jupyter-governance/teams/jupyter-foundation-governing-board) in a comment.

Reviewer: ```suggestion
   - Ping the [`@jupyter-foundation-governing-board` team](https://github.com/orgs/jupyter-governance/teams/jupyter-foundation-governing-board) in a comment on the proposal tracking issue.
```

   - Set the status to {kbd}`Review`.
1. **Review**: Subcommittees review proposals and recommend next actions.

>    - Read [](#timing-goals).

Reviewer: ```suggestion
   - See [](#timing-goals)
```

   - A few potential outcomes from the review process:
      - **Recommend for funding**.
      - **Request changes**.

>       - **Reject** (for low-quality or submissions not aligned with [our funding priorities](priorities.md)).

Reviewer: ```suggestion
      - **Declined** (for example, proposal is low quality or is not aligned with our [funding priorities](priorities.md)).
```
I've been using "approve/decline" language on the tracking board, etc.

1. **Board Review**: At least one week is given to the Foundation Board to request actions other than approval.
   - If there are no objections after one week, then the funding is approved.
   - If there are objections or requested changes, the subcommittee decides how to proceed.
1. **Sign-Off by LF**: The [Treasurer](#role:treasurer) coordinates with the [LF Program Manager](#role:program-manager) and Accountant for financial validation and compliance.
1. **Execution**: Once approved, funds are disbursed by LF (the specific mechanism and timing will depend on the proposal). Proposal owners are responsible for ensuring the work is executed.
1. **Reporting**. Proposal owners should provide regular updates to the Foundation board, ideally in the form of public blog posts. At least one public blog post should be communicated describing the outcomes of the funding effort.

(timing-goals)=

> ## Exepectations and goals for timing

Reviewer: ```suggestion
## Expectations and goals for timing in the first proposal round
```
